### ISSUE TITLE
Proxy environment variables are not taken into account.

### ISSUE THREAD

**Issue by isarere** [sharptree/vscode-autoscript-deploy#18]
Hello,
Thank you for such a great tool. Recently, I've been using your extension on Linux. It seems that environment variables like http_proxy and https_proxy are not taken into account in your code, which is based on Axios. 

I think having the option to configure a proxy would be an interesting feature, is this feature on your roadmap?

Thanks!

Best regards.

**Comment by jvenhuizen** [comment]
That is a good suggestion and something I can look into.

**Comment by jvenhuizen** [comment]
Reading the documentation for the Axios `http_proxy` it seems that this is just handled by default.  Is it possible your environment variable is not set globally, such that the VS Code process is not picking it up?

With that said, there is also the ability to manually set the proxy, so I think we will add the option to the settings.

**Comment by isarere** [comment]
> Reading the documentation for the Axios `http_proxy` it seems that this is just handled by default. Is it possible your environment variable is not set globally, such that the VS Code process is not picking it up?
> 
> With that said, there is also the ability to manually set the proxy, so I think we will add the option to the settings.

Hello Jason,
Thank you for your response. I believe you're right—when I use VSCode directly from Windows, the OS-level proxy is considered by the autoscript-deploy extension, making manage reachable.

However, most of the time, I use WSL extensions in VSCode (with autoscript-deploy also installed on the WSL side). We’ve configured the HTTP proxy in the .zshrc file within WSL, but it seems that manage is still not reachable.
```shell
export all_proxy=http://localhost:3128
export http_proxy=http://localhost:3128
export https_proxy=http://localhost:3128
```
In any case, I think that having the ability to configure the proxy would be a nice feature. :-)

**Comment by jvenhuizen** [comment]
We have added additional settings for defining the proxy.  If you are using SSL on the connection it uses SSL for the proxy.  Host, port and authentication are all supported for specifying a proxy.

This is available in 1.16.0

**Comment by isarere** [comment]
Thanks @jvenhuizen , I'll try it right now. ;-)
